pkg/converter/cloudevents/cloudevents.go

```diff
@@ -65,14 +65,17 @@ func (ce *CloudEvent) Response(data []byte) ([]byte, error) {
 
 	var body interface{}
 	contentType := "text/plain"
-	body = string(data)
 
-	// check if data is in JSON format
-	if json.Valid(data) &&
-		// "foo" is also JSON
-		bytes.Contains(data, []byte("{")) {
+	switch {
+	case json.Valid(data) &&
+		(bytes.TrimSpace(data)[0] == '{' ||
+			bytes.TrimSpace(data)[0] == '['):
 		contentType = "application/json"
 		body = json.RawMessage(data)
+	default:
+		data = bytes.TrimSpace(data)
+		data = bytes.Trim(data, "\"")
```

Comment:
Just for my curiosity: does it mean the runtime always quotes every response? Because if that's the case the following thing will happen:

`json.Valid(data)` -> true
`bytes.TrimSpace(data)[0] == '{'` -> always false due to the format `"{ ... }"`

Or is it because some languages such as Python return strings quoted?

Reply:
I'm testing on the python runtime and if the function returns the object - it is not quoted, so I think it is the latter case

Reply:
Fingers crossed that this hack works with every language and doesn't destroy the output 😅🤞